plasmapy/dispersion/dispersion_functions.py

```diff
@@ -219,41 +120,17 @@ def plasma_dispersion_func_deriv(
     (-0.484255687717376...+0j)
     >>> plasma_dispersion_func_deriv(-1.52+0.47j)
     (0.165871331498228...+0.445879788059350...j)
-
-    For user convenience
-    `~plasmapy.dispersion.dispersion_functions.plasma_dispersion_func_deriv_lite`
-    is bound to this function and can be used as follows:
-
-    >>> plasma_dispersion_func_deriv.lite(0)
-    (-2+0j)
-    >>> plasma_dispersion_func_deriv.lite(1j)
-    (-0.484255687717376...+0j)
-    >>> plasma_dispersion_func_deriv.lite(-1.52+0.47j)
-    (0.165871331498228...+0.445879788059350...j)
-
     """
-
-    if not isinstance(
-        zeta, (numbers.Integral, numbers.Real, numbers.Complex, np.ndarray, u.Quantity)
-    ):
+    try:
+        return -2 * (1 + zeta * plasma_dispersion_func(zeta))
+    except u.UnitTypeError as wrong_units:
+        raise u.UnitsError(
```

Comment:
Coverage on tests is missing here

Reply:
Thanks! I'll add a test for this before merging.